librz/analysis/arch/arm/arm_il64.c

```diff
@@ -1708,7 +1708,7 @@ static RzILOpEffect *movn(cs_insn *insn) {
 static RzILOpEffect *msr(cs_insn *insn) {
 	cs_arm64_op *op = &insn->detail->arm64.operands[0];
 #if CS_API_MAJOR > 4
-	if (op->type != ARM64_OP_SYS || op->sys != ARM64_SYSREG_NZCV) {
+	if (op->type != ARM64_OP_SYS || (ut64)op->sys != (ut64)ARM64_SYSREG_NZCV) {
```

Comment:
Can you please extract this change in a separate PR so that it can be merged in `dev`?

Reply:
But this should only be needed because of the capstone-next change, if I remember correctly

Reply:
Right, nevermind then. I simply forgot.